src/components/Header/Header.jsx

```diff
@@ -10,8 +10,9 @@ import {
 } from 'carbon-components-react/lib/components/UIShell';
 import AppSwitcher from '@carbon/icons-react/lib/app-switcher/20';
 import PropTypes from 'prop-types';
-import React, { useState, useCallback } from 'react';
+import React, { useState } from 'react';
 import { settings } from 'carbon-components';
+import cn from 'classnames';
```

Comment:
We should standardize on this import name. Some places its `cx`, some places it's `classNames`, others `classnames`, this would be another one.

I've opened issue https://github.com/IBM/carbon-addons-iot-react/issues/799 to address